src/server/console/logind_console_services.cpp

```diff
@@ -205,8 +208,8 @@ connect_to_system_bus(mir::GLibMainLoop& ml)
     {
         auto error_msg = error ? error->message : "unknown error";
         BOOST_THROW_EXCEPTION((
-            std::runtime_error{
-                "Failed to find address of DBus system bus: "s + error_msg}));
+                                  std::runtime_error{
+                                      "Failed to find address of DBus system bus: "s + error_msg}));
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style